### PR TITLE
<feature> add breakdown card

### PR DESCRIPTION
**Pull request by shdn-ibm**

### CODE REVIEW

File: src/components/breakdown-card/breakdown-capacity.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as React from 'react';
+import './breakdown-card.scss';
+//import * as classNames from 'classnames';
+
+export const TotalCapacityBody: React.FC<TotalCapacityBodyProps> = ({ value, className }) => {
+  return <p className='capacity-breakdown-card__capacity-body'>{value}</p>;
+};
```
Comment (SanjalKatiyar) [inline src/components/breakdown-card/breakdown-capacity.tsx:22]: `className` is passed, but, never used in the FC:
```
  return (
    <p className={classNames('capacity-breakdown-card__capacity-body', className)}>
      {value}
    </p>
  );
 ```
 Is it used as a placeholder or something ?

File: src/components/capacity-breakdown/capacity-breakdown-card.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+import './capacity-breakdown-card.scss';
+import { humanizeBinaryBytes } from '../../humanize';
+import { BreakdownQueryMapODF } from '../../constants/queries';
+import { PROJECTS, STORAGE_CLASSES, PODS } from '../../constants/index';
+import { 
+  getInstantVectorStats,
+} from '../../selectors/promethues-utils';
+import { parseProps } from '../../selectors/index';
+
+const keys = [PROJECTS, STORAGE_CLASSES, PODS];
```
Comment (SanjalKatiyar) [inline src/components/capacity-breakdown/capacity-breakdown-card.tsx:38]: nits (not anything important): variable name as `dropdownKeys` or `dropdownItems` could be used for better readability.

File: src/components/capacity-breakdown/capacity-breakdown-card.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    query: queries[queryKeys[1]],
+    endpoint: "api/v1/query" as any,
+  });
+  const metricTotal = _.get(totalUsedmetric, 'data.result[0].value[1]');
+
+  const [usedmetric] = usePrometheusPoll({
+    query: queries[queryKeys[2]],
+    endpoint: "api/v1/query" as any,
+  });
+  const flashsystemUsed =  _.get(usedmetric, 'data.result[0].value[1]');
```
Comment (SanjalKatiyar) [inline src/components/capacity-breakdown/capacity-breakdown-card.tsx:64]: just a thought, can we use something similar to this, will this work ??
```
queryKeys.forEach((q) => usePrometheusPoll({
    query: queries[q],
    endpoint: "api/v1/query" as any,
  });

const results = queryKeys.map((q) => {
     const value = _.get(q, 'data.result[0].value[1]');
     return value ? value : q;
});
```

File: src/components/capacity-breakdown/capacity-breakdown-card.tsx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+            isCheckboxSelectionBadgeHidden
+          >
+            {breakdownSelectItems}
+          </Select>
+        </div>
+      </DashboardCardHeader>
+      <DashboardCardBody className="flashsystem-capacity-breakdown-card__body">
+        <BreakdownCardBody
+          isLoading={false}
+          hasLoadError={false}
```
Comment (SanjalKatiyar) [inline src/components/capacity-breakdown/capacity-breakdown-card.tsx:99]: Is there a specific reason for assigning `false` to these props ??
We can poll for it as well right ?
```
const [metrics, metricsLoadError, metricsLoading] = usePrometheusPoll({ ... });
```

File: src/components/capacity-breakdown/capacity-breakdown-card.tsx (hunk 3 above)
Comment (shdn-ibm) [reply src/components/capacity-breakdown/capacity-breakdown-card.tsx]: Tried, the metric result is undefined.

Comment (SanjalKatiyar) [reply src/components/capacity-breakdown/capacity-breakdown-card.tsx]: ack !! I am not sure why it is undefined, we poll similarly in our console code too.
Okay, these things can be refactored later also I guess, if need arises.

**Comment by SanjalKatiyar** [comment]
reviewed the code, it LGTM
/assign @bipuladh

**Review by bipuladh — APPROVED** [review]

**Pull request merged**